web_widget_child_selector/models/base.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, _
+
+
+class Base(models.AbstractModel):
+    _inherit = 'base'
+
+    def get_record_parent(self):
+        self.ensure_one()
+        if not hasattr(self, self._parent_name):
+            return False
+        parent = getattr(self, self._parent_name)
+        if not parent:
+            return False
+        return parent.name_get()[0]
+
+    def _get_record_parents(self, field):
+        if not self:
+            return [(False, _('Root'))]
```

Comment:
I guess it is used to be able to set the field to False when using this widget but I think it can cause confusion since the word Root doesnt have anything to do with setting a field to false

Reply:
Maybe a good option is to add a X icon next to the name when in edit mode that sets the field to false and we can get rid off that 'Root' element.